### ISSUE TITLE
The script isn't working

### ISSUE THREAD

**Issue by Mikede78** [C0nw0nk/qBittorrent#2]
I updated the script with how many days to delete but its not doing anything when I run it. Can you help?

**Comment by Tmp341** [comment]
Can you create another batch file, and paste this inside?
``qbt.cmd > batch.log 2>&1``

Then run the batch file. Afterwards, check log file to see the error.

**Comment by Mikede78** [comment]
I keep getting the message below. I tried a restart and then ran th
bat file again still got the same message.

The process cannot access the file because it is being used by another
process.

C:\qbt>

**Comment by Tmp341** [comment]
Have you run it as admin?

**Comment by Mikede78** [comment]
Yes I did. I attached a video of it. In case it doesn't work, I'm sharing
it through google drive.

https://drive.google.com/file/d/1MRGF5yQgkjRDiZqUkUh-YHqsuzmMaHVB/view?usp=sharing

**Comment by Tmp341** [comment]
I meant as "run qbt.cmd as admin" 😃 , still this shows something else is running files.

**Comment by Mikede78** [comment]
Sorry for the confusion. So you want me to run the bat file with qbt.cmd >
batch.log 2>&1, first and then run the qbt.cmd?

**Comment by Tmp341** [comment]
> Sorry for the confusion. So you want me to run the bat file with qbt.cmd > batch.log 2>&1, first and then run the qbt.cmd?

You ran the bat file and saw "The process cannot access the file because it is being used by another
process."

You are trying to run a batch file (qbt.cmd) under C drive, which Windows' is asking you via UAC (or whatever it is now), you might be hitting something qbt.cmd can't bypass. This is "last update 2y ago" script, might be anything.

**Comment by Mikede78** [comment]
Yeah the UAC only prompts when something requires admin promissions.

Can you explain how to use the logging in more detail?

**Comment by Tmp341** [comment]
My knowledge about scripts is limited. You can try [LockHunter](https://lockhunter.com/download.htm) or any other app you can find to see which process is using the file.

**Comment by Mikede78** [comment]
Ok no problem. Do you have the script running?

**Comment by Tmp341** [comment]
> Ok no problem. Do you have the script running?

Not for some time.
